tests/test_kubernetes_tools.py

```diff
@@ -2816,8 +2816,19 @@ def test_max_unavailable(instances, bmf):
 
 
 def test_pod_disruption_budget_for_service_instance():
+    mock_kube_deploy_config = KubernetesDeploymentConfig(
+        service="my-service",
+        instance="my-instance",
+        cluster="cluster",
+        config_dict={},
+        branch_dict=None,
+    )
+    mock_namespace = mock_kube_deploy_config.get_namespace()
```

Comment:
ohh I think these tests will fail if I set it to ``mock_namespace="fake_namespace"`` because for example in this test:
```
def test_create_pod_disruption_budget():
    mock_client = mock.Mock()
    mock_pdr = V1beta1PodDisruptionBudget()
    mock_namespace = "fake_namespace"
    create_pod_disruption_budget(mock_client, mock_pdr, mock_namespace)
    mock_client.policy.create_namespaced_pod_disruption_budget.assert_called_with(
        namespace="paasta", body=mock_pdr
    )
```
We check to see in ``assert_called_with`` if we are passing the namespace "paasta".

Reply:
I don't mean to change the behaviour of the unit tests in this PR. I think we can also probably get away with not passing ``mock_namespace`` to the functions that has ``namespace: str = "paasta"`` since it's an optional argument. I wanted to be more clear so I passed the mock_namespace but we can also not pass it